app/views/leagues/ranking.html.erb

```diff
@@ -34,3 +34,9 @@
     <% end %>
   </tbody>
 </table>
+
+<%# this didnt work here %>
+<%# app/helpers/button_helper.rb %>
+<%= back_btn %>
+<%# this did, kind of reimplementing the feature but what do you say?%>
+<a class="btn btn-default" href="javascript:history.back()" ><%= t('helpers.links.back') %></a>
```

Comment:
does `javascript:history.back()` not imitate the  brower's back button, which could potentially lead to a user being navigated out of our system? I know that back buttons in other places of Sportify already work this (wroooong) way.
What was the prob with `+<%= back_btn %>` ?

Reply:
The  `<%= back_btn %>`  is actually doing the same thing but when this view is loaded the back button points to the view itself. What do you think the back button should be doing? Or maybe thats a question for our PO. @svitlanakalancha What do you think?

Reply:
Another suggestion: how about adding a specific target for the back button? Something like this: `<%= link_to t('helpers.links.back'), league_path(@event), class: 'btn btn-default' %>`

Reply:
To my mind, the back button should always simply point to the corresponding league's page. This would imply that ”Move back“ means something like ”Move one layer up in the hierarchical layering of our app” aka “Move up”. I think this is a common practice for back buttons on those kinds of applications – however, asking our PO sounds like a good idea. 👍

Reply:
yup, i agree. that'd then lead to the back button actually just being a conventional link in disguise, i would assume.

Reply:
The back button should lead to the correspondent event page as the story says. I agree with @bakoe opinion. We could rename it to 'Zurück zur Liga' or 'Zurück zum Turnier', but I don't think it is necessary. User can open the table only from event page, so it may be understandable, what happens by clicking 'Zurück' :)